GoogleCombined/Podfile

```diff
@@ -0,0 +1,13 @@
+source 'https://github.com/CocoaPods/Specs.git'
+target 'GoogleProject' do
```

Comment:
The target name should also be the same name as the sub-directory (i.e. `MapsAndPlacesDemo`).